### DISCUSSION TITLE
Strange drag&drop on sharing menu

### DISCUSSION THREAD

**Discussion by Alwaysin** [FreshRSS/FreshRSS#6455] [General]
![chrome_ykjfH84wjc](https://github.com/FreshRSS/FreshRSS/assets/7189936/6af83bda-638c-4e16-9749-d80ab9d1718a)

It throws an error when doing it:
```
Uncaught TypeError: Cannot read properties of undefined (reading 'remove')
    at HTMLFormElement.<anonymous> (draggable.js?1695387681:85:10)
(anonymous) @ draggable.js?1695387681:85
```

It seems you cannot upload a file but you have the UI to do so when dragging a file from the file explorer to there.

The banner says "configuration has been updated" ^_^

Probably no problem but just wanted to share in case there could be security problem, or bad data insertion in the database or whatever.

**Comment by math-GH** [comment]
The drag&drop feature needs a bit stealing :D 
Same here: https://github.com/FreshRSS/FreshRSS/issues/6449
And the user query drag&drop has this issue too

**Comment by math-GH** [comment]
@Alwaysin Could you please test #6505?

**Reply by Alwaysin** [reply]
Will do when merged in edge if possible!

**Reply by Alkarex** [reply]
Edge is our rolling release, so it is nice to test before. But that should be easy to do. How do you install FreshRSS (git, Docker, ZIP...)?

**Comment by Alwaysin** [comment]
It works
